config/defaults.ini.php

```diff
@@ -0,0 +1,487 @@
+; Two-letters ISO 639-1 code.
+language = en
+
+; like_no_comment (Boolean)
+; Don't update the "commented" value of an item when it is liked.
+like_no_comment = false
+
+; local_block (Boolean)
+; Used in conjunction with "block_public".
+local_block = false
+
+; local_search (Boolean)
+; Blocks search for users who are not logged in to prevent crawlers from blocking your system.
+local_search = false
+
+; local_tags (Boolean)
+; If activated, all hashtags will point to the local server.
+local_tags = false
+
+; max_connections (Integer)
+; The maximum number of database connections which can be in use before the worker process is deferred to its next interval.
+; When the system can't detect the maximum numbers of connection then this value can be used. Use 0 for auto-detection.
+max_connections = 0
+
+; max_connections_level (Integer 0-100)
+; The maximum percentage of connections that are allowed to let the worker start.
+max_connections_level = 75
+
+; max_contact_queue (Integer)
+; Maximum number of queue items for a single contact before subsequent messages are discarded.
+max_contact_queue = 500
+
+; max_batch_queue (Integer)
+; Maximum number of batched queue items for a single contact before subsequent messages are discarded.
+max_batch_queue = 1000
+
+; max_image_length (Integer)
+; An alternate way of limiting picture upload sizes.
+; Specify the maximum pixel  length that pictures are allowed to be (for non-square pictures, it will apply to the longest side).
+; Pictures longer than this length will be resized to be this length (on the longest side, the other side will be scaled appropriately).
+; If you don't want to set a maximum length, set to -1.
+max_image_length = -1
+
+; max_processes_backend (Integer)
+; Maximum number of concurrent database processes for background tasks.
+max_processes_backend = 5
+
+; max_processes_frontend (Integer)
+; Maximum number of concurrent database processes for foreground tasks.
+max_processes_frontend = 20
+
+; maximagesize (Integer)
+; Maximum size in bytes of an uploaded photo.
+maximagesize = 800000
+
+; min_poll_interval (Integer)
+; minimal distance in minutes between two polls for a contact. Reasonable values are between 1 and 59.
+min_poll_interval = 1
+
+; no_regfullname (Boolean)
+; Allow pseudonyms (true) or enforce a space between firstname and lastname in Full name, as an antispam measure (false).
+no_regfullname = true
+
+; optimize_max_tablesize (Integer)
+; Maximum table size (in MB) for the automatic optimization.
+; -1 to disable automatic optimization.
+;  0 to use internal default (100MB)
+optimize_max_tablesize = -1
+
+; session_handler (database|cache|native)
+; Whether to use Cache to store session data or to use PHP native session storage.
+session_handler = database
+
+; cache_driver (database|memcache|memcached|redis)
+; Whether to use Memcache or Memcached or Redis to store temporary cache.
+cache_driver = database
```

Comment:
I'm missing the `lock_driver` configuration

Reply:
ok, It's because it got chosen automatically and not necessary in the default.ini.php .
So you can ignore it ..

Reply:
It should be set in the `defaults.ini.php` file though, I probably missed a lot of implicit/recently added config keys.